avocado/utils/process.py

```diff
@@ -538,6 +538,13 @@ def stop(self):
             self.terminate()
         return self.wait()
 
+    def get_pid(self):
+        """
+        Reports PID of this process
+        """
+        self._init_subprocess()
```

Comment:
Well decorator or custom `__getattr__` would be IMO better, but so far we'd been using this...